account_move_tag/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Account Move Tag",
+    "summary": """
+        Adds tags to the account move so vendor bills and customer invoices can be
+        easily distinguished in the accounting journal.
+    """,
+    "author": "Codeforward B.V., Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-invoicing",
+    "category": "Hidden",
+    "version": "17.0.1.0.0",
+    "depends": ["account"],
+    "license": "LGPL-3",
```

Comment:
Why not putting it `AGPL-3` for avoiding private extensions on this?

Reply:
Hi Pedro, we'll dive into the licensing question ;)